src/components/VHeader/VHeaderMobile.vue

```diff
@@ -1,45 +1,43 @@
 <template>
   <header
-    class="main-header z-30 flex w-full items-center justify-between gap-x-2 gap-y-4 bg-white px-4 py-3 md:items-stretch md:py-4 md:px-7"
+    class="main-header z-30 flex w-full items-center justify-between gap-x-2 bg-white px-4 py-3 md:items-stretch md:py-4 md:px-7"
     :class="{
-      'flex-wrap md:flex-nowrap': !isHeaderScrolled,
       'border-b border-white': !isHeaderScrolled && !isMenuOpen,
-      'border-b border-dark-charcoal-20':
-        isSearchRoute && (isHeaderScrolled || isMenuOpen),
-      'md:justify-start': !isSearchRoute,
-      'flex-nowrap': !isSearchRoute && isHeaderScrolled,
+      'border-b border-dark-charcoal-20': isHeaderScrolled || isMenuOpen,
     }"
   >
     <VLogoButton :is-fetching="isFetching" :is-search-route="isSearchRoute" />
 
     <VSearchBar
```

Comment:
There are some height inconsistencies in the mobile header.
<img width="365" alt="Screenshot 2022-09-29 at 11 54 08 AM" src="https://user-images.githubusercontent.com/16580576/192973678-e26ab4ef-060d-4ad0-8ee0-04aff6ce09d3.png">

Reply:
Thank you, @dhruvkb ! I did not try to fix the mobile header, as you say, because it's not in scope of this PR.